Tests/Unit/UrlParserTests.swift

```diff
@@ -0,0 +1,24 @@
+import Quick
+import Nimble
+@testable import MiniApp
+
+class UrlParserTests: QuickSpec {
+    override func spec() {
+        describe("Url parser") {
+            context("when parseForFileDirectory is called with valid url") {
+                it("will returns the path to store the file") {
+                    let urlString = "https://www.example.com/version/rak0123/img/onload/file.png"
+                    let path = UrlParser.parseForFileDirectory(with: urlString)
+                    expect(path).toEventually(equal("img/onload/file.png"))
+                }
+            }
+            context("when parseForFileDirectory is called with invalid url") {
+                it("will return nil") {
+                    let urlString = "https://www.example.com/mini-ver/rak0123/img/onload/file.png"
+                    let path = UrlParser.parseForFileDirectory(with: urlString)
+                    expect(path).toEventuallyNot(equal("img/onload/file.png"))
```

Comment:
why not `expect(path).toEventually(beNil())` ?

Reply:
Well, the Urlparser doesn't seem to be returning nil for now. Like I said in the PR description, this class need to refactored to update the latest endpoints and most of these things can be avoided.

Reply:
Oh, I just was confused by `it("will return nil")`, sorry